### ISSUE TITLE
mini-media-player not rounded when using coverart

### ISSUE THREAD

**Issue by syphernl** [basnijholt/lovelace-ios-dark-mode-theme#3]
According to the theme (and mini-media-player) it should become rounded. However, for some reason this isn't happening for me.

Is there anything specific required to get it rounded?

**Comment by basnijholt** [comment]
You can see my configuration files [here](https://github.com/basnijholt/home-assistant-config).

I didn't do anything special though (AFAIK).

P.S. don't forget to star ⭐️ this repo 😃

**Comment by syphernl** [comment]
> You can see my configuration files [here](https://github.com/basnijholt/home-assistant-config).
> 
> I didn't do anything special though (AFAIK).

Nope, doesn't look like you have anything specific for that card. Odd, it keeps on being square here 🤔 

> P.S. don't forget to star ⭐️ this repo 😃

I just did ;-)

**Comment by basnijholt** [comment]
Did you try restarting your HA a few times?

**Comment by syphernl** [comment]
Yeah. And incognito mode. What I however do see happening onload is that it is rounded. 
But then it suddenly changes to square once it adds the fanart.

I have it configured like this:
```
  - type: conditional
    conditions:
      - entity: media_player.music
        state: "playing"
    card:
      type: custom:mini-media-player
      entity: media_player.music
      info: scroll
      artwork: cover
      hide:
        icon: true
        power: true
        volume: true
        shuffle: false
```

It is not caused by the `conditional` since I use the same config on a different page without `conditional`. It looks like the artwork is the cause, especially since you do not use it like that in your setup.

**Comment by basnijholt** [comment]
Ah, this is probably a bug in `mini-media-player`, @kalkih what do you think?

**Comment by syphernl** [comment]
It looks that way. I've created an issue on the `mini-media-player` project: https://github.com/kalkih/mini-media-player/issues/190

**Comment by basnijholt** [comment]
Great that you figured it out. We can keep this issue open for others to see until kalkih/mini-media-player#190 is fixed.

**Comment by kalkih** [comment]
Quoting myself from the other issue 😆 

> Hey,
> This should work out of the box, was actually fixed in the latest release [v1.4.2](https://github.com/kalkih/mini-media-player/releases/tag/v1.4.2).
> Make sure you're not loading an older version of the card from cache, try manually clearing the browser cache.
> 
> If you're using HACS, please check [kalkih/mini-graph-card/issues/169](https://github.com/kalkih/mini-graph-card/issues/169), same information is relevant for this card.

**Comment by syphernl** [comment]
Reinstalling mini-media-player solved the issue.
